src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -90,72 +90,75 @@ public static EmailGenerator inst() {
                 ? instructorsLogic.getInstructorsForCourse(session.getCourseId())
                 : new ArrayList<>();
```

Comment:
Can we get coowner from instructors in `generateFeedbackSessionEmailBases`?

Reply:
I would say it's possible